kubernetes/samples/charts/apache-webtier/README.md

```diff
@@ -63,11 +66,14 @@ installing the chart. For example:
 ```console
 $ helm install my-release --values values.yaml apache-webtier
 ```
+## useNonPriviledgedPorts
+By default, the chart will install the Apache webtier on PriviledgedPort i.e port 80. You need to have the flag `useNonPriviledgedPorts=true` to enable the Apache webtier to listen on port `8080`
```

Comment:
PriviledgedPort i.e port 80. -> PriviledgedPort, for example, port 80.
You need to have the flag -> Set the flag

Reply:
Updated. Since the port used is 80 only added inside bracket.